test/integration/gpu_rays.cc

```diff
@@ -597,7 +597,8 @@ void GpuRaysTest::RaysParticles(const std::string &_renderEngine)
       abs(box02Pose.Pos().Y()) - unitBoxSize / 2;
 
   // set a larger tol for particle range
-  double laserNoiseTol = particleSize.X();
+  // depth noise is computed based on particle size
+  double laserNoiseTol = particleSize.X() + particleSize.X() * 0.5;
```

Comment:
This test is failing now:

<details>
  <summary>GpuRays/GpuRaysTest.RaysParticles</summary>

```
  [ RUN      ] GpuRays/GpuRaysTest.RaysParticles/ogre2
  /github/workspace/test/integration/gpu_rays.cc:618: Failure
  Value of: particleHit || particleMiss
    Actual: false
  Expected: true
  actual vs expected particle range: 1.3223873376846313 vs 1
  /github/workspace/test/integration/gpu_rays.cc:618: Failure
  Value of: particleHit || particleMiss
    Actual: false
  Expected: true
  actual vs expected particle range: 1.3650697469711304 vs 1
  /github/workspace/test/integration/gpu_rays.cc:618: Failure
  Value of: particleHit || particleMiss
    Actual: false
  Expected: true
  actual vs expected particle range: 1.375098705291748 vs 1
  /github/workspace/test/integration/gpu_rays.cc:618: Failure
  Value of: particleHit || particleMiss
    Actual: false
  Expected: true
  actual vs expected particle range: 1.3158919811248779 vs 1
  /github/workspace/test/integration/gpu_rays.cc:618: Failure
  Value of: particleHit || particleMiss
    Actual: false
  Expected: true
  actual vs expected particle range: 1.3633447885513306 vs 1
  /github/workspace/test/integration/gpu_rays.cc:618: Failure
  Value of: particleHit || particleMiss
    Actual: false
  Expected: true
  actual vs expected particle range: 1.3735580444335938 vs 1
  /github/workspace/test/integration/gpu_rays.cc:618: Failure
  Value of: particleHit || particleMiss
    Actual: false
  Expected: true
  actual vs expected particle range: 1.3367263078689575 vs 1
  /github/workspace/test/integration/gpu_rays.cc:618: Failure
  Value of: particleHit || particleMiss
    Actual: false
  Expected: true
  actual vs expected particle range: 1.3569036722183228 vs 1
  /github/workspace/test/integration/gpu_rays.cc:618: Failure
  Value of: particleHit || particleMiss
    Actual: false
  Expected: true
  actual vs expected particle range: 1.3486930131912231 vs 1
  /github/workspace/test/integration/gpu_rays.cc:618: Failure
  Value of: particleHit || particleMiss
    Actual: false
  Expected: true
  actual vs expected particle range: 1.3585772514343262 vs 1
  /github/workspace/test/integration/gpu_rays.cc:618: Failure
  Value of: particleHit || particleMiss
    Actual: false
  Expected: true
  actual vs expected particle range: 1.338756799697876 vs 1
  /github/workspace/test/integration/gpu_rays.cc:618: Failure
  Value of: particleHit || particleMiss
    Actual: false
  Expected: true
  actual vs expected particle range: 1.3856438398361206 vs 1
  /github/workspace/test/integration/gpu_rays.cc:618: Failure
  Value of: particleHit || particleMiss
    Actual: false
  Expected: true
  actual vs expected particle range: 1.3853719234466553 vs 1
  /github/workspace/test/integration/gpu_rays.cc:618: Failure
  Value of: particleHit || particleMiss
    Actual: false
  Expected: true
  actual vs expected particle range: 1.3508977890014648 vs 1
  /github/workspace/test/integration/gpu_rays.cc:618: Failure
  Value of: particleHit || particleMiss
    Actual: false
  Expected: true
  actual vs expected particle range: 1.3497778177261353 vs 1
  /github/workspace/test/integration/gpu_rays.cc:618: Failure
  Value of: particleHit || particleMiss
    Actual: false
  Expected: true
  actual vs expected particle range: 1.3797192573547363 vs 1
  /github/workspace/test/integration/gpu_rays.cc:618: Failure
  Value of: particleHit || particleMiss
    Actual: false
  Expected: true
  actual vs expected particle range: 1.3519102334976196 vs 1
  /github/workspace/test/integration/gpu_rays.cc:618: Failure
  Value of: particleHit || particleMiss
    Actual: false
  Expected: true
  actual vs expected particle range: 1.3346030712127686 vs 1
  [  FAILED  ] GpuRays/GpuRaysTest.RaysParticles/ogre2, where GetParam() = "ogre2" (9448 ms)
```  

</details>

Any guidance on how this should be ported forward, @iche033 ?

Reply:
ran into this test failure in https://github.com/ignitionrobotics/ign-rendering/pull/269 and should be fixed by that PR

Reply:
#269 is now included in this PR